tee-worker/litentry/core/assertion-build/src/dynamic/contracts/token_holding_amount/TokenQueryLogic.sol

```diff
@@ -57,57 +45,48 @@ abstract contract TokenQueryLogic is TokenHoldingAmount {
 			.identityToString(network, identity.value);
 
 		if (identityToStringSuccess) {
-			string memory url;
-			uint32[] memory networks = tokenNetworks[tokenName];
 			uint256 totalBalance = 0;
 
-			for (uint32 i = 0; i < networks.length; i++) {
```

Comment:
Removing the for loop here, it should not be possible to query all network addresses of a token, I guess.